crates/cargo-util-schemas/src/manifest/mod.rs

```diff
@@ -165,6 +165,15 @@ pub struct InheritablePackage {
     pub rust_version: Option<RustVersion>,
 }
 
+#[derive(Deserialize, Serialize, Clone, Debug)]
+#[serde(untagged, rename_all = "kebab-case")]
+pub enum TomlPackageBuild {
+    /// If there's just one script or boolean value
+    SingleScript(StringOrBool),
+    /// If multiple build script are there
+    MultipleScripts(Vec<String>),
+}
```

Comment:
Personally, I'd suggest
```suggestion
#[derive(Deserialize, Serialize, Clone, Debug)]
#[serde(untagged, rename_all = "kebab-case")]
pub enum TomlPackageBuild {
    Auto(bool)
    /// If there's just one script or boolean value
    SingleScript(String),
    /// If multiple build script are there
    MultipleScripts(Vec<String>),
}
```

Reply:
I think it should be fine now?

Reply:
Yes, `TomlPackageBuild` now uses `Auto`, thanks!

Could you split adding `MultipleScripts` out into its own commit?

Reply:
I have just added and set to `unimplemented!`.

Should I remove it and then do it again?

Reply:
While doing experiments (which you pushing your draft pr will end up including) it is fine for the history to be dirty.  However, for complete chunks of code (rename `readme`, forking `StringOrBool`), it would be good to clean up the history for those.

A clean history is important for understanding a change.  The commits in a PR should tell a story with each part small enough that it is trivial to look at.

If you need help in cleaning up any of the history, ping me on Zulip on we can get on a call.